testsuite/features/secondary/srv_advanced_search.feature

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2022 SUSE LLC.
+# Licensed under the terms of the MIT license.
+
+@scope_monitoring @sle_minion @sle_client
```

Comment:
We were always having a new line for each tag, but I guess it's fine like that.

Reply:
nah, it's fine, I'll change it to match